core/src/main/java/edu/ucr/cs/riple/core/Config.java

```diff
@@ -403,10 +410,21 @@ public Config(String configPath) {
             : Paths.get(nullawayLibraryModelLoaderPathString);
     moduleInfoList.remove(0);
     this.downstreamInfo = ImmutableSet.copyOf(moduleInfoList);
+    this.moduleCounterID = 0;
     this.log = new Log();
     log.reset();
   }
 
+  /**
+   * Returns the latest id associated to a module, used to create unique ids for each module and
+   * increments it.
+   *
+   * @return last id value used in integer.
+   */
+  public int getNextModuleUniqueID() {
```

Comment:
I see no uses of this method outside this class itself. Given that, it should be private, but, since it additionally is a one-liner, why aren't we just using `moduleCounterID++` in the two places we call `getNextModuleUniqueID()` and just removing this method?

Reply:
This method is called in the `scanner.out.MethodInfo.java` in the first line of the constructor. Do you still think I should remove this method ?

Reply:
How? This is a method of the `Config` class within `core`... the method called inside `MethodInfo` in scanner is `ScannerContext::getNextUniqueID()`

Reply:
Thank you for the clarification, I mistakenly thought you are talking about `getNextUniqueID` in Scanner config. Changed the visibility from public to private. https://github.com/nimakarimipour/NullAwayAnnotator/commit/26083ada5e91d7ef4b8cc3ab3bbe994c51c68599